.drone.star

```diff
@@ -274,6 +274,14 @@ def uploadCoverage(ctx):
           'mc mirror cache/cache/%s/%s/coverage coverage/' % (ctx.repo.slug, ctx.build.commit + '-${DRONE_BUILD_NUMBER}'),
         ]
       },
+      {
+        'name': 'show-before',
+        'image': 'owncloud/alpine:latest',
+        'pull': 'always',
+        'commands': [
+          'cat coverage/*',
```

Comment:
Do you want to keep this "debugging" or or not?

Reply:
removed